error.md

```diff
@@ -0,0 +1,62 @@
+# Errors faced during migration with nikita v1
+
+## Installation of semver necessary
+
+When running the `bootstrap` command, I got an error stating that `semver` module was not found.
+
+### Solution
+
+npm install semver
```

Comment:
If you don't use semver in your code, then this is a quick fix, not a solution

Reply:
Ok